tests/test_packageAlerts.py

```diff
@@ -469,25 +467,26 @@ def testRun_without_produce(self, mock_server_check):
         doWriteAlerts set to true.
         """
         packConfig = PackageAlertsConfig(doWriteAlerts=True)
-        tempdir = tempfile.mkdtemp(prefix='alerts')
```

Comment:
And again here, at least there was a `shutil.rmtree()`, but using the context manager is better.